UnitedIncome/components
# Atoms/icons/arrow-ellipsis-icon -> TypeScript

### Description
> Please provide a brief description of your changes below.
convert one file to TS just to make sure I'm doing it right

### Testing Instructions
> What should be done in order to test this pull request, or re-create its effects.
check to make sure conversion was done right.

### Screenshots
> If this pull request includes a visual change please attach any applicable screenshots.

### Comments
> Any additional comments that will aid with the review process.
